public/js/data.js

```diff
@@ -70,9 +70,11 @@ async function updateValue(metricId) {
     return await $.get("/api/tickerMetric/" + metricId + "/" + ticker);
 }
 
-function updatingChart(period) {
+//updating the Price Chart display
+function updatingChart(period, chartType, btnElemID) {
```

Comment:
I believe this function only need to take in period as a single argument. If this is the case could you please delete chartType, and btnElemID?
Do the same in the html handlebar template please

Reply:
Done.